### PR TITLE
Fix bug with HA deployment with v1

### PR DESCRIPTION
**Pull request by rukawata**
HA deployment in v1 currently fails due to two issues with SUSE image:
1. hostname not set stable (issue #395)
2. SSH key overwritten for ha cluster configuration

With this PR we fixes:
1. Ensure the correct hostname are set and not changed for SLES
2. Exchange the same key on all hosts (primary node, secondary node and iscsi device)
3. Enable cloud config since we are now running on SLES12 SP5

Tests done:
HA E2E deployment completed.

### CODE REVIEW

File: deploy/vm/ansible/roles/ssh-key-distribute/tasks/main.yml [hunk 1]
```diff
@@ -1,44 +1,15 @@
...
+    creates: /root/.ssh/id_rsa
 
-- debug:
-    var: user_info
+- name: Ensure the Public SSH keys are available for exchanging SSH key trust between nodes
+  shell: cat /root/.ssh/id_rsa.pub
+  register: cluster_public_ssh_key
 
-- name: Add authorized key to remote
+- name: Ensure the Primary Node public key is authorized on all nodes, required for crm_clustering
```
Comment (tniek) [inline deploy/vm/ansible/roles/ssh-key-distribute/tasks/main.yml:10]: What do you mean by crm_clustering? crm_mon?

File: deploy/vm/ansible/roles/ssh-key-distribute/tasks/main.yml [hunk 2]
```diff
@@ -1,44 +1,15 @@
...
-    ssh_key_bits: 2048
-  register: user_info
+- name: Ensure there are SSH keys for the root user to communicate between nodes
+  shell: crm cluster init -y ssh
+  args:
+    creates: /root/.ssh/id_rsa
 
-- debug:
-    var: user_info
+- name: Ensure the Public SSH keys are available for exchanging SSH key trust between nodes
```
Comment (tniek) [inline deploy/vm/ansible/roles/ssh-key-distribute/tasks/main.yml:6]: "public"

File: deploy/vm/ansible/roles/hotfix/tasks/main.yml [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+---
+
+- name: Ensure correct hostnames are set and not changed for SLES
+  become: true
+  become_user: root
+  when:
+    - ansible_os_family|upper == 'SUSE'
+  block:
+
+  #SUSE bug 1167134
```
Comment (tniek) [inline deploy/vm/ansible/roles/hotfix/tasks/main.yml:10]: nit: "# SUSE"

File: deploy/vm/ansible/roles/ssh-key-distribute/tasks/main.yml (hunk 1 above)
Comment (rukawata) [reply deploy/vm/ansible/roles/ssh-key-distribute/tasks/main.yml:10]: Sorry, I meant crm cluster, meaning crm cluster init/join.

**Pull request merged**